azext_iot/commands.py

```diff
@@ -121,6 +121,12 @@ def load_command_table(self, _):
         cmd_group.command('show', 'iot_dps_registration_get')
         cmd_group.command('delete', 'iot_dps_registration_delete')
 
+    with self.command_group('iotcentral app', command_type=iotcentral_ops) as cmd_group:
```

Comment:
Wasn't there a plan to include aliases for these commands? One's that are uniform with the rest of the IoT command space? As in `az iotcentral app` should have an alias `az iot central app`, with the latter being the go forward (`az iotcentral` to be deprecated as soon as possible?)

Reply:
These were in another branch as to not clutter this PR, but I have merged it in now